survey-runner-queue/cloudwatch.tf

```diff
@@ -0,0 +1,61 @@
+resource "aws_cloudwatch_metric_alarm" "rabbitmq1_cluster_status" {
+  alarm_name          = "${var.env}-rabbitmq1-cluster-alert"
+  evaluation_periods  = "1"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
```

Comment:
Isn't this the binary status that is either 1 or 0? so won't it always equal 1 when its alive?